sunbird-es-utils/src/main/resources/reindexing/mappings/certreg_mappings.json

```diff
@@ -1,156 +1,248 @@
+      "type":"boolean",
+      "fields":{
+        "raw":{
+          "type":"boolean"
         }
       }
     },
-    "jsonUrl": {
-      "type": "text",
-      "fields": {
-        "raw": {
-          "type": "text",
-          "fielddata": true
+    "jsonUrl":{
+      "type":"text",
+      "fields":{
+        "raw":{
+          "type":"text",
+          "analyzer":"keylower",
+          "fielddata":true
         }
       },
-      "copy_to": [
+      "copy_to":[
         "all_fields"
       ],
-      "analyzer": "cs_index_analyzer",
-      "search_analyzer": "cs_search_analyzer",
-      "fielddata": true
+      "analyzer":"cs_index_analyzer",
+      "search_analyzer":"cs_search_analyzer",
+      "fielddata":true
     },
-    "pdfUrl": {
-      "type": "text",
-      "fields": {
-        "raw": {
-          "type": "text",
-          "fielddata": true
+    "pdfUrl":{
+      "type":"text",
+      "fields":{
+        "raw":{
+          "type":"text",
+          "analyzer":"keylower",
+          "fielddata":true
         }
       },
-      "copy_to": [
+      "copy_to":[
         "all_fields"
       ],
-      "analyzer": "cs_index_analyzer",
-      "search_analyzer": "cs_search_analyzer",
-      "fielddata": true
+      "analyzer":"cs_index_analyzer",
+      "search_analyzer":"cs_search_analyzer",
+      "fielddata":true
     },
-    "reason": {
-      "type": "text",
-      "fields": {
-        "raw": {
-          "type": "text",
-          "fielddata": true
+    "reason":{
+      "type":"text",
+      "fields":{
+        "raw":{
+          "type":"text",
+          "analyzer":"keylower",
+          "fielddata":true
         }
       },
-      "copy_to": [
+      "copy_to":[
         "all_fields"
       ],
-      "analyzer": "cs_index_analyzer",
-      "search_analyzer": "cs_search_analyzer",
-      "fielddata": true
+      "analyzer":"cs_index_analyzer",
+      "search_analyzer":"cs_search_analyzer",
+      "fielddata":true
     },
-    "recipient": {
-      "properties": {
-        "id": {
-          "type": "text"
+    "recipient":{
+      "properties":{
+        "id":{
+          "type":"text",
+          "fields":{
+            "raw":{
+              "type":"text",
+              "analyzer":"keylower",
+              "fielddata":true
+            }
+          },
+          "copy_to":[
+            "all_fields"
+          ],
+          "analyzer":"cs_index_analyzer",
+          "search_analyzer":"cs_search_analyzer",
+          "fielddata":true
         },
-        "type": {
-          "type": "text"
+        "type":{
+          "type":"text",
+          "fields":{
+            "raw":{
+              "type":"text",
+              "analyzer":"keylower",
+              "fielddata":true
+            }
+          },
+          "copy_to":[
+            "all_fields"
+          ],
+          "analyzer":"cs_index_analyzer",
+          "search_analyzer":"cs_search_analyzer",
+          "fielddata":true
+        },
+        "name":{
```

Comment:
name, email and phone were intentionally kept out of search queries to prevent misuse. Please remove them.

Reply:
done.